src/points/PointsHistoryBottomSheet.test.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react'
+import { fireEvent, render, waitFor } from '@testing-library/react-native'
+import PointsHistoryBottomSheet from 'src/points/PointsHistoryBottomSheet'
+import { createMockStore, RecursivePartial } from 'test/utils'
+import { RootState } from 'src/redux/reducers'
+import { FetchMock } from 'jest-fetch-mock/types'
+import { Provider } from 'react-redux'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { PointsEvents } from 'src/analytics/Events'
+import { getHistoryStarted } from 'src/points/slice'
+import { GetHistoryResponse } from 'src/points/types'
+
+jest.mock('@gorhom/bottom-sheet', () => {
+  const react = require('react-native')
+  return {
+    __esModule: true,
+    default: react.View,
+    BottomSheetSectionList: react.SectionList,
```

Comment:
can you add this to `__mocks__/@gorhom/bottom-sheet.ts` instead so that it's applied globally?